pkg/apis/noderecovery/v1alpha1/types.go

```diff
@@ -54,6 +68,10 @@ type NodeRemediationStatus struct {
 	Reason string `json:"reason,omitempty"`
 	// Phase indicates NodeRemediation recovery state
 	Phase NodeRemediationPhase `json:"phase,omitempty"`
+	// StartTime indicates when current phase started
+	StartTime metav1.Time `json:"startTime,omitempty"`
+	// ElapsedTime indicates how many time, object has the same phase
+	ElapsedTime time.Duration `json:"elapsedTime,omitempty"`
```

Comment:
Wouldn't you want this to be calculated (otherwise it will always be wrong) ?

Reply:
I will remove this field, I added it just to initiate an update on the object, but I will just re-enqueue object.